package.json

```diff
@@ -9,9 +9,8 @@
   },
   "scripts": {
     "lint": "eslint src",
-    "test": "npm run lint",
```

Comment:
Doesn't feel like we should remove linting

Reply:
oops didn't see that was removed. I'll re-add it

Reply:
so it looks like the test command was removed but linting is still there. Added precommit linting